docs/ref/signals.txt

```diff
@@ -217,6 +217,11 @@ instance. Strictly speaking, this is not a model signal since it is sent by the
 :data:`pre_save`/:data:`post_save` and :data:`pre_delete`/:data:`post_delete`
 when it comes to tracking changes to models, it is included here.
 
+.. versionadded:: 1.10
```

Comment:
1.11 and move to the bottom of the "action" section where the other additions are made